assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { InputGroup, SelectMenu } from '../../form';
+import { MAX_FONT_SIZE, MIN_FONT_SIZE } from '../../../constants';
+import { useFont } from '../../../app/font';
+
+function FontControls({ properties, state, setState }) {
+  const { fontFamily, fontWeight, fontSize } = properties;
+  const {
+    state: { fonts },
+    actions: { getFontWeight, getFontFallback },
+  } = useFont();
+
+  const handleNumberChange = useCallback(
+    (property) => (value) =>
+      setState({ ...state, [property]: parseInt(value) }),
+    [setState, state]
+  );
+  return (
+    <>
+      {fonts && (
+        <SelectMenu
+          label={__('Font family', 'web-stories')}
+          options={fonts}
+          value={state.fontFamily}
+          isMultiple={fontFamily === ''}
+          onChange={(value) => {
+            const currentFontWeights = getFontWeight(value);
+            const currentFontFallback = getFontFallback(value);
+            const fontWeightsArr = currentFontWeights.map(
+              ({ thisValue }) => thisValue
```

Comment:
```suggestion
              ({ value }) => value
```
Looks like a typo.

Reply:
`value` is already used above, it's actually not a typo. It's brought over from the previous fonts panel.

Reply:
Hmm, are you looking at my Suggested change or your code? 😉

Reply:
What I'm saying is that we can't change it to `({ value }) => value` here since `value` variable is already defined above in the code (can't define same name twice). Or did you have something else in mind?

Reply:
If the value inside the object is called `thisValue`, we can't really rename it. So my comment is really about the naming required for the dropdown (and the lists generated by the font provider) using the weird name `thisValue` for something that should probably just be named `value`.

If and when the list internal is renamed to `value`, this map could just be `.map(({ value: v }) => v);` to avoid the name collision. But not before the list and dropdown is changed to use `value` and not `thisValue`. All that's for future optimization though.

Reply:
Now I get it, I was too focused on the bug to see the `no-shadow` rule.

So it should be:
```suggestion
              ({ value: thisValue }) => thisValue
```
or `weight => weight.value` if you don't like renaming.

We were both right 😃